frames/section.lua

```diff
@@ -274,9 +311,12 @@ function sectionFrame:_DoCreate()
     GameTooltip:Hide()
   end)
 
-  title:SetScript("OnClick", function()
+  title:RegisterForClicks("RightButtonUp")
+
+  title:SetScript("OnClick", function(_, e)
     if s.headerDisabled then return end
     onTitleClickOrDrop(s)
+    if (e == "RightButton") then onTitleRightClick(s) end
```

Comment:
You need to check for left or right click here, i.e.

```lua
if e == "LeftButton" then
 -- do left
elseif e == "RightButton" then
 -- do right
end
```

Also, removed unneeded parentheses.

Reply:
Do you 'drop' flow happens based on LeftButton as well?

Reply:
Yes, it's okay to make the drop flow only happen on left click.